README.md

```diff
@@ -9,6 +9,18 @@ _Part of the [Solana Mobile Stack](https://github.com/solana-mobile/solana-mobil
 
 Join us on [Discord](https://discord.gg/solanamobile)
 
+## See our [documentation website](https://docs.solanamobile.com/) for:
+- [General Development Setup](https://docs.solanamobile.com/getting-started/development_setup)
+- React Native
+    - [Quickstart Setup](https://docs.solanamobile.com/react-native/quickstart)
+    - [dApp Integration Guide](https://docs.solanamobile.com/react-native/mwa_integration_rn)
+    - [Hello World Tutorial](https://docs.solanamobile.com/getting-started/hello_world_tutorial)
+- Android
+    - [Quickstart Setup](https://docs.solanamobile.com/android-native/quickstart)
+    - [dApp Integration Guide](https://docs.solanamobile.com/android-native/mwa_integration)
+- [Sample App Reference](https://docs.solanamobile.com/sample-apps/sample_app_overview)
```

Comment:
nit: do we want to be consistent on calling them "dApp" vs "app"? I personally don't care actually, but maybe others have strong opinions

Reply:
that's a good point and we should be standardizing the language across all our docs. I can update to `dApp` in this PR and make docsite changes too.

On a similar note, I was discussing with @Funkatronics was the naming between `dApp` vs `DApp` and we landed on `dApp`.

Reply:
Deciding to leave these as `dApp` instead of `app`. I think the term `dApp` is useful in making a distinction between dApps and Wallet Apps. This is a distinction that's made in the [MWA protocol spec](https://solana-mobile.github.io/mobile-wallet-adapter/spec/spec.html#dapp-identity-verification) and its useful.